app/vep/vep_resources.py

```diff
@@ -88,3 +88,40 @@ async def vep_status(request: Request, submission_id: str):
     except Exception as e:
         logging.debug(e)
         return response_error_handler(result={"status": 500})
+
+@router.get("/download/{submission_id}", name="download_results")
```

Comment:
We haven't written a spec; but following the template of our other VEP endpoints, this should probably be `/submissions/:id/download` within the `/api/tools/vep` namespace